templates/admin/hook_new.tmpl

```diff
@@ -22,6 +22,8 @@
 					<img class="img-13" src="{{StaticUrlPrefix}}/img/dingtalk.ico">
 				{{else if eq .HookType "telegram"}}
 					<img class="img-13" src="{{StaticUrlPrefix}}/img/telegram.png">
+				{{else if eq .HookType "teamcity"}}
+					<img class="img-13" src="{{StaticUrlPrefix}}/img/teamcity.png">
```

Comment:
There is a SVG version available: https://commons.wikimedia.org/wiki/File:TeamCity_Icon.svg

Reply:
✔